docs/sections/user_guide/uw_yaml/forecast_yaml.rst

```diff
@@ -0,0 +1,232 @@
+.. _forecast_yaml:
+
+Forecast YAML
+=============
+
+The structured YAML to run a forecast is described below. It is enforced via JSON Schema.
+
+In this section, there are entries that define where the forecast will run (``run_directory:``), what will run (``executable:``), the data files that should be staged (``cycle-dependent:`` and ``static:``), and blocks that correspond to the configuration files required by the forecast model (``fd_ufs:``, ``field_table:``, ``namelist:``, etc.). Each of the configuration file blocks will allow the user to set a template file as input, or to define a configuration file in its native key/value pair format with an option to update the values (``update_values:``) contained in the original input file (``base_file:``).
+
+The configuration files required by the ufs-weather-model are documented :weather-model-io:`here<model-configuration-files>`.
+
+The ``forecast:`` section
+-------------------------
+
+The ``forecast:`` section describes the specifics of the FV3 atmosphere forecast component.
+
+.. code-block:: yaml
+
+  forecast:
+    cycle-dependent:
+      INPUT/gfs_data.nc: /path/to/gfs_data.nc
+      INPUT/sfc_data.nc: /path/to/sfc_data.nc
+      INPUT/gfs_ctrl.nc: /path/to/gfs_ctrl.nc
+    diag_table:
+      template_file: /path/to/diag_table/template/file
+    domain: regional
+    executable: fv3.exe
+    fd_ufs:
+      base_file: /path/to/base/fd_ufs.yaml
+    field_table:
+      base_file: /path/to/field_table.yaml
+      update_values:
+        liq_wat:
+          longname: cloud water mixing ratio
+          units: kg/kg
+          profile_type:
+            name: fixed
+            surface_value: 2.0
+    length: 12
+    model_configure:
+      base_file: /path/to/base/model_configure
+      update_values:
+        write_dopost: .false.
+    namelist:
+      base_file: /path/to/base/input.nml
+      update_values:
+        fv_core_nml:
+          k_split: 2
+          n_split: 6
+    nems.configure:
+      template_file: /path/to/template/nems.configure
+    run_dir: /path/to/forecast/run/directory/{yyyymmddhh}
+    static:
+      fv3.exe: /path/to/executable/ufs_model
+      INPUT/grid_spec.nc: /path/to/grid_spec.nc
+      ...
+      data_table: /path/to/data_table
+      eta_micro_lookup.data: /path/to/noahmptable.dat
+      noahmptable.tbl: /path/to/noahmptable.tbl
+
+.. _updating_values:
+
+Updating Values
+^^^^^^^^^^^^^^^
+
+Many of the sections describe the configuration files needed by the FV3 Forecast model, i.e. namelist, fd_ufs, model_configure. The ``base_file:`` entry is required so that the file will be staged. Modifications to the base file can be made at run time by providing an ``update_values:`` block.
```

Comment:
```suggestion
Many of the sections describe configuration files needed by FV3, e.g. ``namelist:``, ``fd_ufs:``, ``model_configure:``. The ``base_file:`` section is required to initially stage the file; it can then be modified via an ``update_values:`` block.
```
There's a bit of variance throughout this page in terminology -- `block`, `entry`, `section`. It would be helpful to harmonize them so that readers can count on a clear, unambiguous vocabulary.

Reply:
Just to make sure I am clear on those, here's what I've been assuming: 

`block` corresponds to a nested dict in Python terminology, `entry` is a single key/value pair, and `section` is a top-level `block`.

Reply:
I think the highest-level block can just be called "block" -- I don't think it's ambiguous to say "the `forecast` block" or "the `workflow` block". Even if those keys happen to be repeated somewhere else in a huge UW YAML key, I think it will be clear from context which one we're talking about.